### PR TITLE
Update docs-preview link

### PR DESCRIPTION
**Pull request by nkammah**
Following the migration from Jenkins to Buildkite, docs previews are now available at <repo>_bk_<PR>.

More context in https://github.com/elastic/docs/pull/2898

## Related issues
Closes #ISSUE

### CODE REVIEW

File: resources/github-comment-markdown.template [hunk 1]
```diff
@@ -17,7 +17,7 @@
...
+[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](${jobUrl}/pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](${jobUrl}/tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](${jobUrl}/changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](${jobUrl}/artifacts) <% if(env?.OTEL_ELASTIC_URL != null && !env?.OTEL_ELASTIC_URL.equals('null')) {%>[![preview](https://img.shields.io/badge/elastic-observability-blue)](${env?.OTEL_ELASTIC_URL.replaceAll(' ','+')})<%}%><% if (statusSuccess && snapshoty) {%>[![Snapshots](https://img.shields.io/badge/snapshots-artifacts-blue)](${snapshotyUrl})<%}%>
```
Comment (v1v) [inline resources/github-comment-markdown.template:20]: What's the reason for this change? IIUC, the change in the groovy script is the only required one

```suggestion
[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](${jobUrl}/pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](${jobUrl}/tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](${jobUrl}/changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](${jobUrl}/artifacts) <% if(docsUrl?.trim() && !buildStatus?.equals('ABORTED')) {%>[![preview](https://img.shields.io/badge/docs-preview-yellowgreen)](${docsUrl})<%}%>  <% if(env?.OTEL_ELASTIC_URL != null && !env?.OTEL_ELASTIC_URL.equals('null')) {%>[![preview](https://img.shields.io/badge/elastic-observability-blue)](${env?.OTEL_ELASTIC_URL.replaceAll(' ','+')})<%}%><% if (statusSuccess && snapshoty) {%>[![Snapshots](https://img.shields.io/badge/snapshots-artifacts-blue)](${snapshotyUrl})<%}%>
```

**Review by v1v — CHANGES_REQUESTED** [review]
template contains a change that won't be needed

**Review by v1v — APPROVED** [review]

**Review by amannocci — APPROVED** [review]
